### DISCUSSION TITLE
HMT1800

### DISCUSSION THREAD

**Discussion by Hennesder2te** [tbnobody/OpenDTU#245] [Ideas]
Ich habe einen HMT1800 (dreiphasiger Wechselrichter).
Den kann ich nicht über die DTU auslesen. Ist da eine erweiterung möglich/geplant?
LG KHB

**Comment by tbnobody** [comment]
Zuerst müsste man herausfinden welches Funkmodul für die HMT Serie zu verwenden ist. (Hier wird ein Modul mit einer Frequenz von <1GHz verwendet). Anschließend müsste sich jemand finden, der einen HMT Inverter und eine Original DTU hat. Derjenige müsste wohl die Platine analysieren, und anschließend die Pakete zwischen MCU und RF Modul mitsniffen... und  am Schluss die Pakete analysieren und interpretieren um auf die entsprechenden Payloads Commands und Payloads zu schließen.
